applications/FSIApplication/tests/FSI_problem_emulator_test.py

```diff
@@ -255,10 +401,63 @@ def ComputeDirichletNeumannResidual(self):
 
         return disp_residual
 
+    def __ComputeInitialResidual(self):
+        # Solve the fluid emulator problem with the initial displacement guess
+        self.__SolveFluidEmulator()
+
+        # Transfer fluid load
+        for i in range(len(self.corrected_iteration_value_f)):
+            self.corrected_iteration_value_f[i] = self.iteration_value_f[i]
+        self.__TransferFluidEmulatorLoad()
+
+        # Solve structure problem
+        self.structure_solver.SolveSolutionStep()
+
+        # Compute the displacement residual
+        disp_residual = self.__ComputeStructureResidual()
+
+        return disp_residual
+
+    def __SolveFluidEmulator(self):
+        # Calculate the spring reactions that emulate the fluid load over the structure interface
+        for i in range(self.GetInterfaceProblemSize()):
+            self.iteration_value_f[2*i] = self.fluid_emulator_stiffness * self.corrected_iteration_value_u[2*i]
+            self.iteration_value_f[2*i+1] = self.fluid_emulator_stiffness * self.corrected_iteration_value_u[2*i+1]
+
+    def __TransferFluidEmulatorLoad(self):
+        interface_submodelpart = self.structure_solver.main_model_part.GetSubModelPart("StructureInterface2D_Solid_interface")
+
+        i = 0
+        for node in interface_submodelpart.Nodes:
+            point_load = Vector(3)
```

Comment:
just saying that you could also pass a `list` instead

Reply:
As mentioned above this test needs to be completely refactored. I'd leave these changes for the a standalone PR doing this.